Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 group :development do
   gem 'bigdecimal' # necessary on ruby-3.3+
   gem 'bundler', '>= 1.16', '< 3'
+  gem 'fiddle' # necessary on ruby-3.5+
```

Comment:
This seems to break JRuby and TruffleRuby in CI, so could you use `platforms: %i[mri windows]`?

Reply:
Thanks, I added a platforms argument!